Dockerfile

```diff
@@ -1,35 +1,35 @@
-FROM ubuntu-debootstrap:14.04
+FROM php:7.0-alpine
 MAINTAINER Christian Lück <christian@lueck.tv>
 
-RUN DEBIAN_FRONTEND=noninteractive apt-get update && apt-get install -y \
-  nginx supervisor php5-fpm php5-cli \
-  php5-pgsql php5-mysql php5-sqlite php5-mssql \
-  wget
+## Install php and dependencies
+RUN apk --no-cache add postgresql-dev sqlite-dev \
+    && docker-php-ext-configure pdo_pgsql -with-pgsql=/usr/include/postgresql/ \
+    && docker-php-ext-configure pdo_sqlite \
+    && docker-php-ext-install pdo pdo_sqlite pdo_mysql pdo_pgsql
 
-# add adminer as the only nginx site
-ADD adminer.nginx.conf /etc/nginx/sites-available/adminer
-RUN ln -s /etc/nginx/sites-available/adminer /etc/nginx/sites-enabled/adminer
-RUN rm /etc/nginx/sites-enabled/default
+## php odbc hack (@see https://github.com/docker-library/php/issues/103)
+RUN set -x \
+    && apk --no-cache add unixodbc-dev \
+    && cd /usr/src/php/ext/odbc \
+    && phpize \
+    && sed -ri 's@^ *test +"\$PHP_.*" *= *"no" *&& *PHP_.*=yes *$@#&@g' configure \
+    && ./configure --with-unixODBC=shared,/usr \
+    && docker-php-ext-install odbc
 
-# install adminer and default theme
-RUN mkdir /var/www
-RUN wget http://www.adminer.org/latest.php -O /var/www/index.php
-RUN wget https://raw.github.com/vrana/adminer/master/designs/hever/adminer.css -O /var/www/adminer.css
-WORKDIR /var/www
-RUN chown www-data:www-data -R /var/www
+## Add Tini
+RUN apk add --no-cache --repository http://dl-cdn.alpinelinux.org/alpine/edge/community/ tini
+ENTRYPOINT ["/usr/bin/tini", "--"]
 
-# tune PHP settings for uploading large dumps
-RUN echo "upload_max_filesize = 2000M" >> /etc/php5/upload_large_dumps.ini \
- && echo "post_max_size = 2000M"       >> /etc/php5/upload_large_dumps.ini \
- && echo "memory_limit = -1"           >> /etc/php5/upload_large_dumps.ini \
- && echo "max_execution_time = 0"      >> /etc/php5/upload_large_dumps.ini \
- && ln -s ../../upload_large_dumps.ini /etc/php5/fpm/conf.d \
- && ln -s ../../upload_large_dumps.ini /etc/php5/cli/conf.d
+## Add the files
+ADD php.ini /usr/local/etc/php/conf.d/php.ini
 
-# expose only nginx HTTP port
-EXPOSE 80
+## install adminer and default theme
+RUN mkdir -p /var/www
+ADD http://www.adminer.org/latest.php /var/www/index.php
```

Comment:
We should create tags for this image, that are in sync with adminer (e.g. `clue/adminer:4.2.4`). So instead of `latest` we should point to the specific version `https://www.adminer.org/static/download/4.2.4/adminer-4.2.4.php`